packages/checkout/README.md

```diff
@@ -1,123 +1,182 @@
-<div align="center">
-  <p align="center">
-    <a href="https://docs.x.immutable.com/docs">
-      <img src="https://cdn.dribbble.com/users/1299339/screenshots/7133657/media/837237d447d36581ebd59ec36d30daea.gif" width="280"/>
-    </a>
-  </p>
-  <h1>Immutable Checkout SDK</h1>
-</div>
+# Checkout
 
-# Overview
+## Table of Contents
 
-> **Warning** **IMMUTABLE CHECKOUT SDK IS UNSTABLE** <br/>
-> Since it has not hit the version 1.0 yet, its public interface should not be considered final. Future releases may include breaking changes without further notice. We will do our best to keep this documentation updated providing visibility on breaking changes planned.
+- [Pre-requisites](#pre-requisites)
+- [Checkout SDK Quick Start Guide](#checkout-sdk-quick-start-guide)
+- [Widgets Library Quick Start Guide](#widgets-library-quick-start-guide)
+  - [Widgets Development Locally](#widgets-development-locally)
+- [Project structure](#project-structure)
 
-The Immutable Checkout SDK provides both programatical and user interfaces for interacting with various wallets and L2 networks. It provides all the tools to enable transaction ready gamers including on ramping, dex swaps, bridging and more.
+### Pre-requisites
 
-## Documentation
+Install dependencies for the workspace
 
-- Coming Soon &trade;
-
-# Project structure
-
-```
-/
-│
-└───sdk -> Checkout SDK
-│
-└───sdk-sample-app --> Sample app where the Checkout SDK is consumed
-│
-└───widgets-lib --> Checkout widgets
-│
-└───widgets-sample-app --> Sample app where the Checkout Widgets are consumed
-│
-└───README.md
+```bash
+yarn install
 ```
 
-# Running locally
+### Checkout SDK Quick Start Guide
 
-## Dependencies
+All commands below need to be run in the context of the `checkout-sdk` package unless specified otherwise. Read more about context [here](../../../README.md#context).
 
-CheckoutSDK has been built with and tested using the following dependencies.
+Running in `dev` mode using the `immutable-devnet` network:
 
-| Dependency | Version |
-| ---------- | ------- |
-| Node       | v16+    |
-| Yarn       | v3.5.0  |
+```bash
+yarn start:dev
+```
 
-Node Module dependencies are still added to the package.json of each checkout module, however each module installs it's dependencies to a shared node_modules directory at the root level of the Unified SDK. This requires dependency versions to be aligned across all SDK submodules.
+Running in `sandbox`/`production` mode using the `immutable-testnet`/`mainnet` network based on the environment value set for the `SANDBOX_CONFIGURATION` config object in the `sdk.ts` file:
```

Comment:
this is the `src/sdk.ts` file, correct?
the doc mentioned "everything said below will be in context of checkout/sdk folder" so I wasn't sure if there suppose to be another sdk.ts 😆